websites/Y/YouTube Music/presence.ts

```diff
@@ -60,7 +66,7 @@ presence.on("UpdateData", async () => {
 
 	if (["playing", "paused"].includes(mediaSession.playbackState)) {
 		if (privacyMode) {
-			presenceData.type = ActivityType.Listening;
+			presenceData.type = ActivityType.Playing;
```

Comment:
```suggestion
			presenceData.type = ActivityType.Listening;
```
We have a global setting to use playing status over of any other.
Please keep the listening type.